resources/lang/fr/disposable.php

```diff
@@ -3,78 +3,75 @@
     // Common
     'actions'     => 'Actions',
     'airport'     => 'Aéroport',
-    'all'         => 'Voir Tous',
+    'all'         => 'Afficher tout',
     'altn'        => 'Altn',
     'alternate'   => 'Alternatif',
     'close'       => 'Fermer',
-    'darkmode'    => 'Mode Sombre',
+    'darkmode'    => 'Mode Nuit',
     'dest'        => 'Dest',
     'destination' => 'Destination',
-    'fuel_ob'     => 'Carburant à Bord',
-    'latest'      => 'Derniers',
+    'fuel_ob'     => 'Fuel embarqué',
+    'latest'      => 'Récents',
     'map'         => 'Carte',
-    'notes'       => 'Remarques',
     'orig'        => 'Orig',
     'origin'      => 'Origine',
     'registered'  => 'Enregistré',
-    'show_hide'   => 'Voir / Cacher',
+    'show_hide'   => 'Carte / Liste',
     'total'       => 'Total',
     'weather'     => 'Météo',
-    'pagination'  => 'afficher :first premier au :last du total :total des enregistrements',
+    'pagination'  => 'Rapports :first à :last sur :total soumis',
     // Flights
-    'bid_add'     => 'Ajouter une Enchère',
-    'bid_rem'     => 'Retirer une Enchère',
+    'bid_add'     => 'Ajouter aux projets',
+    'bid_rem'     => 'Retirer des projets',
```

Comment:
Since a "projet" is a "project" I don't think this is a good for a bid. Maybe "Ajouter une réservation" / "Retirer la réservation" ?